File: HwStore.Backend/HwStore.Domain/Basket.cs
namespace HwStore.Domain;

public class Basket
{
    public Basket()
    {
        new List<BasketItem>();
    }
    public int Id { get; set; }
    public string? BuyerId { get; set; } = null!;
    public List<BasketItem>? BasketItems { get; set; }

}

File: client/src/app/model/Basket.ts
export interface Basket {
  id: number;
  buyerId: string;
  basketItems: BasketItem[];
}
export interface BasketItem {
  productId: number;
  name: string;
  price: number;
  imageUrl: string;
  quantity: number;
  quantityInBasket: number;
}
